web/middleware.py

```
@@ -141,3 +142,56 @@ def __call__(self, request):
             # Report to Sentry
             sentry_sdk.capture_exception(e)
             return self.get_response(request)
+
+
+class QuizSecurityMiddleware:
+    def __init__(self, get_response):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)